base/abstractarray.jl

```diff
@@ -1152,7 +1182,8 @@ end
     _cshp(tail(dims), (out..., next), tail(shape), tail(nshape))
 end
 
-_cs(d, concat, a, b) = concat ? (a + b) : (a == b ? a : throw(DimensionMismatch(string("mismatch in dimension ", d, " (expected ", a, " got ", b, ")"))))
+_cs(d, concat, a, b) = concat ? (a + b) : (a == b ? a : throw(DimensionMismatch(string(
+    "mismatch in dimension ", d, " (expected ", a, " got ", b, ")"))))
```

Comment:
I'd probably move the entire `throw` to this line. Looks a little funny otherwise.